es/lessons/basics/basics.md

```diff
@@ -66,7 +66,7 @@ iex> 0x1F
 
 ### Coma flotante
 
-En Elixir, los números con coma flotante requieren un decimal después de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
+En Elixir, los números con coma flotante requieren un decimal antes de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
```

Comment:
¿a qué se debe este cambio? el texto original en inglés es:
> floating point numbers require a decimal **after** at least one digit

Reply:
```suggestion
En Elixir, los números con coma flotante requieren un decimal después de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
```

Reply:
That did not go as expected 😞 
![image](https://user-images.githubusercontent.com/73386/68907628-be783000-0705-11ea-8e15-fcf98c6e3f70.png)

I will merge this and push a commit with your change.